client/shared/interfaces/locale_records_interface.coffee

```diff
@@ -14,6 +14,10 @@ module.exports = class LocaleRecordsInterface extends BaseRecordsInterface
       onSuccess: (response) ->
         if response.ok
           response.text().then (data) ->
-            eval(data)
+            # some, not all (why?), moment.js locale files have return
+            # statements on the end of them, but they are not wrapped in
+            # functions which raises an error.
+            # So we stip the return statement with this regex.
+            eval data.replace(/return.+\n\n$/, "\n")
```

Comment:
I'd suggest doing this in client/tasks/shared.coffee, so we just have to do it once on build, rather than on every request